### PR TITLE
Update README.md and CHANGELOG.md

### PR DESCRIPTION
**Pull request by wasky**

### CODE REVIEW

**Comment by wasky** [comment]
> We also need to keep the old one as well.

Is it related to #511 and you want to keep Java support?

**Comment by burhanrashid52** [comment]
> > We also need to keep the old one as well.
> 
> Is it related to #511 and you want to keep Java support?

Yes

**Comment by wasky** [comment]
I would introduce new API before the release rather than keep the old one in docs. The reason for this is the exception handling. In current old version of API and in AsyncTask version of API you can't handle potential exceptions from the lib. In new API this will work:

```
try {
    photoEditor.saveBitmap()
} catch(e: Exception) {
    //
}
```

I made a PR with new API for Java (#512) and changed the README in this PR to show example usage of new API for borh Java and Kotlin.

File: CHANGELOG.md [hunk 1]
```diff
@@ -84,4 +84,11 @@ new TextStyleBuilder()
...
 ### 2.0.0
-- New : Migrated the app and library to Kotlin
\ No newline at end of file
+- New : Migrated the app and library to Kotlin
+
+### 3.0.0
+- New : Arrow shape
+- Change : (Breaking Change) `minSdkVersion` changed to `21`
+- Change : (Breaking Change) Shape names are no longer UPPERCASE
+- Change : `saveAsFile(String, [SaveSettings,] OnSaveListener)` and `saveAsBitmap([SaveSettings,] OnSaveBitmap)` are deprecated use `saveAsFile(String[, SaveSettings])` and `saveAsBitmap([SaveSettings])`
```
Comment (burhanrashid52) [inline CHANGELOG.md:93]: We need to revert the deprecation part.

File: README.md [hunk 2]
```diff
@@ -212,14 +212,26 @@ It will take default fonts provided in the builder. If we want different Emoji f
...
-  
-  
-  
 
 ## Saving
-   
-   We need to provide a file with callback method when edited image is saved
-   
+
+The code below saves an image to a file.
```
Comment (burhanrashid52) [inline README.md:218]: We need to mention the "In v.3.0.0 onward, we can save an image to a file using coroutines:" Need to link v.3.0.0 tag.

**Comment by burhanrashid52** [comment]
@wasky I've published the jar. Can you update the documentation?
Also, do you have a Twitter handle?

**Comment by wasky** [comment]
> @wasky I've published the jar. Can you update the documentation?

Done!

> Also, do you have a Twitter handle?

My Twitter username is `@panwasky`

**Review by burhanrashid52 — APPROVED** [review]

**Pull request merged**